examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -64,14 +64,27 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - configmaps
```

Comment:
If nginx is configured with a configmap for custom configuration we need to be able to read that, too.